python/clx/tests/test_port_heuristic.py

```diff
@@ -27,6 +27,7 @@ def test_major_ports():
     expected["conns"] = [2, 1, 1]
 
     actual = ports.major_ports(input_addr_col, input_port_col)
+    actual = actual.sort_values("addr").reset_index(drop=True)
```

Comment:
Is it possible exchange this line for something else that manipulates `expected` instead of `actual` ?
(Perhaps sorting `expected` instead?)

Reply:
`expected` is already sorted so i just updated port heuristic to sort results. used to be sorted by default but now you have to explicitly set `sort` to `True` in groupby.